index.js

```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+//This document consists of the javascript codes for the webpage
```

Comment:
Add dynamic content in this file

Reply:
I will add dynamic content in the coming stages of the project. For now, I have not added anything in this file.